File: Java/Snack.java
import java.util.Scanner;

/**
 * Snack
 */
public class Snack {

    public static void main(String[] args) {
        Scanner raf = new Scanner(System.in);
        int x = raf.nextInt();
        int y = raf.nextInt();
        float[] price = {4.00f,4.50f,5.00f,2.00f,1.50f};
        for(int i = 1; i <= price.length; i++){
            if(i == x){
                System.out.printf("Total: R$ %.2f\n",(y * price[i-1]));
            }
        }
        
    }
}

File: Python/Snack.py
x, y = map(int, input().split(" "))

"""price = [4.00, 4.50, 5.00, 2.00, 1.50]

print("%.2f"%price[1])"""

if x == 3:
    print("Total: R$ %.2f"%(y*5.00))
elif x == 4:
    print("Total: R$ %.2f"%(y*2.00))
elif x == 2:                                 
   print("Total: R$ %.2f"%(y*4.50))
elif x == 1:
   print("Total: R$ %.2f"%(y*4.00))
elif x == 5:
   print("Total: R$ %.2f"%(y*1.50))
